source/extensions/filters/http/ratelimit/ratelimit.cc

```diff
@@ -159,6 +159,8 @@ void Filter::complete(Filters::Common::RateLimit::LimitStatus status,
     cluster_->statsScope().counterFromStatName(stat_names.ok_).inc();
     break;
   case Filters::Common::RateLimit::LimitStatus::Error:
+    const auto request_id_ = request_headers_->get(Http::Headers::get().RequestId)->value().getStringView();
```

Comment:
`get` of `RequestHeaderMap` will not return `HeaderEntry` and there is no guarantee that the request id will be included in the request headers.

Reply:
so maybe we should remove request_id ?

Reply:
May be you can use stream id as a alternative. Or you can use a placeholder if request id does not exist.